modular_skyrat/modules/ambitions/code/ambition.dm

```diff
@@ -324,11 +326,13 @@
 				log_action("--Requested an admin review--", FALSE)
 				message_admins("<span class='adminhelp'>[ADMIN_TPMONTY(usr)] has requested a review of their ambitions. (<a href='?src=[REF(src)];admin_pref=show_ambitions'>VIEW</a>)</span>")
 				message_admins(span_big(span_adminhelp("THIS WILL BE AUTO-APPROVED IN FIVE MINUTES UNLESS YOU <a href='?src=[REF(src)];admin_pref=cancel_autoapp'>CANCEL</a> IT")))
-				for(var/client/I in GLOB.admins)
-					if(I.prefs.toggles & SOUND_ADMINHELP)
-						SEND_SOUND(I, sound('modular_skyrat/modules/admin/sound/duckhonk.ogg'))
-					window_flash(I, ignorepref = TRUE)
-				auto_approve_timerid = _addtimer(CALLBACK(src, .proc/auto_approve), 10 MINUTES, TIMER_UNIQUE|TIMER_CLIENT_TIME)
+				if(!auto_approve_honked)
+					auto_approve_honked = TRUE
+					for(var/client/I in GLOB.admins)
+						if(I.prefs.toggles & SOUND_ADMINHELP)
+							SEND_SOUND(I, sound('modular_skyrat/modules/admin/sound/duckhonk.ogg'))
+				window_flash(I, ignorepref = TRUE)
```

Comment:
```suggestion
				if(!auto_approve_honked)
					auto_approve_honked = TRUE
					for(var/client/I in GLOB.admins)
						if(I.prefs.toggles & SOUND_ADMINHELP)
							SEND_SOUND(I, sound('modular_skyrat/modules/admin/sound/duckhonk.ogg'))
						window_flash(I, ignorepref = TRUE)
```
Otherwise your thing ain't going to work.

Reply:
haha; I fix one thing and break another. Perfectly balanced